catalog/src/main/kotlin/com/adevinta/spark/catalog/configurator/samples/buttons/ButtonsConfigurator.kt

```diff
@@ -0,0 +1,321 @@
+            )
+            val iconsSides = IconSide.values()
+            val iconsSidesLabel = iconsSides.map { it.name }
+            SegmentedButton(
+                options = iconsSidesLabel,
+                selectedOption = iconSide.name,
+                onOptionSelect = { iconSide = IconSide.valueOf(it) },
+                modifier = Modifier
+                    .fillMaxWidth()
+                    .height(48.dp),
+            )
+        }
+        Column {
+            Text(
+                text = "Style",
+                modifier = Modifier.padding(bottom = 8.dp),
+                style = SparkTheme.typography.body2.copy(fontWeight = FontWeight.Bold),
+            )
+            val buttonStyles = ButtonStyle.values()
+            val buttonStylesLabel = buttonStyles.map { it.name }
+            SegmentedButton(
+                options = buttonStylesLabel,
+                selectedOption = style.name,
+                onOptionSelect = { style = ButtonStyle.valueOf(it) },
+                modifier = Modifier
+                    .fillMaxWidth()
+                    .height(48.dp),
+            )
+        }
+
+        val intents = ButtonIntent.values()
+        var expanded by remember { mutableStateOf(false) }
+        SelectTextField(
+            modifier = Modifier.fillMaxWidth(),
+            value = "",
+            onValueChange = {},
+            readOnly = true,
+            label = "Intent",
+            expanded = expanded,
+            onExpandedChange = {
+                expanded = !expanded
+            },
+            onDismissRequest = {
+                expanded = false
+            },
+            dropdownContent = {
+                intents.forEach {
+                    DropdownMenuItem(
+                        text = { Text(it.name) },
+                        onClick = {
+                            intent = it
+                            expanded = false
+                        },
+                    )
+                }
+            },
+        )
+
+        Column {
+            Text(
+                text = "IconSize",
+                modifier = Modifier.padding(bottom = 8.dp),
+                style = SparkTheme.typography.body2.copy(fontWeight = FontWeight.Bold),
+            )
+            val sizes = ButtonSize.values()
+            val sizesLabel = sizes.map { it.name }
+            SegmentedButton(
+                options = sizesLabel,
+                selectedOption = size.name,
+                onOptionSelect = { size = ButtonSize.valueOf(it) },
+                modifier = Modifier
+                    .fillMaxWidth()
+                    .height(48.dp),
+            )
+        }
+
+        TextField(
+            modifier = Modifier.fillMaxWidth(),
+            value = buttonText,
+            onValueChange = {
+                buttonText = it
+            },
+            label = "Button text",
+            placeholder = "Vérifier les Disponibilité",
```

Comment:
Should we stick to English language? 
Also all the strings are hardcoded, I would suggest extracting them so that configuration screen labels can be reused